When using open, use path relative to imagetool-uncompressed.py

Signed-off-by: Andrei Gherzan <andrei@gherzan.ro>

Index: git/mkimage/imagetool-uncompressed.py
===================================================================
--- a/mkimage/imagetool-uncompressed.py
+++ b/mkimage/imagetool-uncompressed.py
@@ -18,7 +18,7 @@ re_line = re.compile(r"0x(?P<value>[0-9a
 mem = [0 for i in range(32768)]
 
 def load_to_mem(name, addr):
-   f = open(name)
+   f = open(os.path.dirname(__file__) + '/' + name)
 
    for l in f.readlines():
       m = re_line.match(l)
